chainservice/chainservice.go

```diff
@@ -230,6 +230,11 @@ func (cs *ChainService) Start(ctx context.Context) error {
 	if err := cs.consensus.Start(ctx); err != nil {
 		return errors.Wrap(err, "error when starting consensus")
 	}
+	if cs.indexBuilder != nil {
```

Comment:
Moving here to build index before blocksync?

Reply:
yes